### PR TITLE
:sparkles: Add persist engine admin cli

### PR DESCRIPTION
**Pull request by Y-Henry**
Add the persist engine admin cli.
Implements the find all changesets execution.
Add 2 methods to persist engine.

### Cli Commands

**./updatarium-admin-cli --help**
```
Usage: standalone [OPTIONS] COMMAND [ARGS]...

Options:
  -h, --help  Show this message and exit

Commands:
  list  List the changeSet executions
```

**./updatarium-admin-cli list --help**
```
Usage: updatarium-admin-cli list [OPTIONS]

  List the changeSet executions

Options:
  --limit INT                      limit the changeSet display count (default
                                   10, min 0, max 1000)
  --skip INT                       skip some changeSets (default 0, min 0)
  -c, --change-set-id TEXT         filter with one specific change set
  -s, --with-status [NOT_EXECUTED|EXECUTE|OK|FAIL]
                                   filter executions with specific status
  -e, --with-error                 filter failed execution
```

### CODE REVIEW

**Comment by PierreLeresteux** [comment]
You should edit the README.md and the CONTRIBUTING.md (https://github.com/saagie/updatarium/blob/master/CONTRIBUTING.md#persist-engine-creation) 

You should try to launch the build gradle, because, you omit to add the license header (a gradle task  for the auto format is also include -> see the tasks list)

**Comment by Y-Henry** [comment]
Licence task run, README.md and CONTRIBUTING.md updated

**Comment by PierreLeresteux** [comment]
I think `findExecutionHistoric` should not be in the PersistEngine ... It's a dedicated function for the standalone ... No need to implement it for each PersistEngine

**Comment by Y-Henry** [comment]
It was just a way to validate the arguments directly into the repository and confirm that every implementations have the same pagination constraint. I was just block by the same signature and have to rename the to-implement part.

File: core/src/test/kotlin/io/saagie/updatarium/persist/TestPersistEngine.kt [hunk 1]
```diff
@@ -40,6 +41,14 @@ class TestPersistEngine : PersistEngine(PersistConfig()) {
         return changeSetUnLocked.first { it.executionId == changeSetId }.status
     }
 
+    override fun findRecentExecutions(limit: Int, skip: Int): List<ExecutionReport> {
```
Comment (ilaborie) [inline core/src/test/kotlin/io/saagie/updatarium/persist/TestPersistEngine.kt:44]: use `= emptyList()`

File: persist-mongodb/src/main/kotlin/io/saagie/updatarium/persist/MongodbPersistEngine.kt [hunk 2]
```diff
@@ -70,6 +71,16 @@ class MongodbPersistEngine(override val configuration: PersistConfig = PersistCo
             }
         }
 
+    override fun findRecentExecutions(limit: Int, skip: Int): List<ExecutionReport> = collection.find()
+        .sort(descending(MongoDbChangeSet::statusDate))
+        .skip(skip)
+        .limit(limit)
+        .mapTo(mutableListOf(), MongoDbChangeSet::toExecutionState)
+
+    override fun executionCount(): Int {
```
Comment (ilaborie) [inline persist-mongodb/src/main/kotlin/io/saagie/updatarium/persist/MongodbPersistEngine.kt:80]: use `= ...`

File: updatarium-admin-cli/src/main/kotlin/io/saagie/updatarium/admincli/Standalone.kt [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+
+class Standalone : CliktCommand(name = "updatarium-admin-cli", printHelpOnEmptyArgs = true) {
+    override fun run() {}
+}
+
+class List : CliktCommand(help = "List the changeSet executions", name = "list") {
+
+    companion object {
+        const val EXECUTION_REPORT_MIN_LIMIT = 0
```
Comment (ilaborie) [inline updatarium-admin-cli/src/main/kotlin/io/saagie/updatarium/admincli/Standalone.kt:38]: maybe could be replaced by an `IntRange`

**Pull request closed without merging**